velox/functions/prestosql/DecimalArithmetic.cpp

```diff
@@ -150,6 +150,42 @@ class Subtraction {
   }
 };
 
+class Multiply {
+ public:
+  template <typename R, typename A, typename B>
+  inline static void
+  apply(R& r, const A& a, const B& b, uint8_t aRescale, uint8_t bRescale) {
+    r.setUnscaledValue(
+        a.unscaledValue() * b.unscaledValue() *
```

Comment:
Should we use the new mul function to suppress the sanitizer?

Reply:
Yes we should use the new `mul` in `DecimalUtil.h`